packages/design-system/src/components/IdleTimeout/IdleTimeoutDialog.tsx

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable react/no-multi-comp */
+import React from 'react';
+import { Dialog } from '../Dialog';
+import { Button } from '../Button';
+
+export interface IdleTimeoutDialogProps {
+  /**
+   * The text for the 'continue session' button in warning dialog.
+   */
+  continueSessionText: string;
+  /**
+   * The heading text for the warning dialog.
+   */
+  heading?: string;
+  /**
+   * The text for the button that ends the session in warning dialog.
+   */
+  endSessionButtonText?: string;
+  /**
+   *
+   */
+  endSessionUrl?: string;
+  /**
+   * The message text for the warning dialog.
+   * Note that using the token `<timeToTimeout>` will be replaced in the message text with the number of minutes until timeout.
+   */
+  message: string | React.ReactNode;
+  /**
+   * Optional function that is called when the user chooses to keep the session alive.
+   * The IdleTimeout component will reset the countdown internally.
+   */
+  onSessionContinue: (...args: any[]) => any;
```

Comment:
Is this optional? Not seeing `?` set.

Reply:
Ope! Not optional, I'll update the documentation